packages/renderer/src/lib/flows/RunFlowModal.svelte

```diff
@@ -0,0 +1,89 @@
+<script lang="ts">
+import { Button, Input } from '@podman-desktop/ui-svelte';
+
+import Dialog from '/@/lib/dialogs/Dialog.svelte';
+import type { FlowInfo } from '/@api/flow-info';
+
+interface Props {
+  showModal: boolean;
+  flow: FlowInfo;
+  onRun: (inputValues: Record<string, string>) => void;
+  onCancel: () => void;
+}
+
+let { showModal = $bindable(), flow, onRun, onCancel }: Props = $props();
+
+let inputValues: Record<string, string> = $state({});
+
+const valid = $derived.by(() => {
+  if (flow.parameters) {
+    return flow.parameters.every(param => inputValues[param.name] && inputValues[param.name].trim() !== '');
+  }
+  return true;
+});
+
+$effect(() => {
+  if (showModal) {
```

Comment:
question: in which case do we have this effect with showModal = false as if showModal is false, it's no longer in DOM/destroyed

but showModal should not be there (previous comment)